client/src/store/reducers/ui/persistent/reader.js

```diff
@@ -1,8 +1,26 @@
-import { combineReducers } from "redux";
+// import { combineReducers } from "redux";
 import colors from "./reader/colors";
 import typography from "./reader/typography";
+import { handleActions } from "redux-actions";
 
-export default combineReducers({
+const initialState = {
```

Comment:
This is where I'm struggling right now.  I want the `initialState` of `reader` to be in the shape of `colors` and `typography` which have their own `initialState`s defined.